README.md

```diff
@@ -104,6 +105,25 @@ audit-ci --critical --report false
 audit-ci
 ```
 
+### Settings file
+
+**audit-ci.json**
+
+```json5
+{
+  // Only use one of ["low": true, "moderate": true, "high": true, "critical": true]
+  low: true,
+  report: true, // [Optional] defaults `true`
+  'package-manager': 'auto', // [Optional] defaults `auto`
+  advisories: [100, 101], // [Optional] defaults `[]`
+  whitelist: ['example1', 'example2'], // [Optional] defaults `[]`
+}
+```
+
+```sh
+audit-ci --settings audit-ci.json
```

Comment:
I think we should recommend using `.json5` as the extension if that's what's being used.

Reply:
_Because the file is loaded using Node's require(), the filename MUST end in `.json` to be interpreted correctly._ [src](https://github.com/yargs/yargs/blob/master/docs/api.md#configobject)

I agree that PR is confusing, as it's invalid to copy-paste the example config file. Perhaps creating a specification section for the config file and adding a plain, no-comment example config would be more appropriate.

Reply:
Ok, I have no strong opinion about the example; maybe just a comment noting not to copy and paste but to use valid json instead would be fine.

Reply:
I went with adding a more explicit specification and using a valid JSON example so that it can be copy-pasted.